server/controllers/entitlement/entitlement.go

```diff
@@ -64,33 +82,85 @@ func NewEntitlementController(context context.Context, db *persistence.Database)
 				log.Warnf("Could not initialize entitlements controller. %v\n", err)
 			}
 		}
-
 		entitlementCtrlInstance = &EntitlementAPIController{
 			ctx:            context,
 			httpRequester:  requester,
 			apiUrl:         url,
 			productCode:    code,
 			subscriptionId: subs,
+			tenantId:       tenant,
 			database:       db,
 		}
 	})
 	return entitlementCtrlInstance
 }
 
+func (controller *EntitlementAPIController) CreateEntitlement(orgId string) {
```

Comment:
Is this function actually creating entitlement or is it just triggering/requesting the entitlement to be created?